src/fast-path.js

```diff
@@ -329,6 +329,10 @@ FPp.needsParens = function(assumeExpressionContext) {
         && name === "object"
         && parent.object === node;
 
+    case "NumericLiteral":
+      return parent.type === "MemberExpression"
+        && isNumber.check(node.value);
```

Comment:
Why do we need the `isNumber` check? Don't we already know that because it's a type of `NumericLiteral`?

Also, can you make sure this works both with babylon and the flow parsers?